api/test/wfl/integration/wgs_pipeline_test.clj

```diff
@@ -1,43 +0,0 @@
-(ns wfl.integration.wgs-pipeline-test
```

Comment:
How could I have written this test file better so we'd judge it worth keeping?

Reply:
This test file has been really useful for me! We can add more tests as we go!

Reply:
It's actually been moved to `test/integration/modules/wgs_test.clj`

Reply:
Ah I gotcha, GitHub's diff didn't recognize the move and you simplified the test enough that I didn't recognize it 😅